server.js

```diff
@@ -21,13 +14,135 @@ const app = express()
 app.use(cors())
 app.use(express.json())
 
-// Start defining your routes here
+// Start defining routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send({
+    Welcome: "Welcome to my Netflix API",
+    documentation: "https://documenter.getpostman.com/view/18068162/UVR4PW9m"
+  })
+})
+
+// route provides all endpoints
+app.get('/endpoints', (req, res) => {
+  res.json({
+    response: listEndpoints(app),
+    success: true
+  })
+})
+
+// route provides a sorted list of all countries that are in netflixData
+app.get('/countries', (req, res) => {
+  const countries = Array.from(new Set(netflixData.map(item => item.country))).sort()
```

Comment:
Could you leave a comment about this line of code? :)

Reply:
Sure :)
The aim with this code is to show a list of all countries in the Netflix-Data without any duplicates. Therefore I am creating a Set (object with unique items). I convert the set into an array afterwards and sort it alphabetically